client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml

```diff
@@ -88,6 +96,11 @@ spec:
                 oneOf:
                 - required: ["snapshotHandle"]
                 - required: ["volumeHandle"]
+              sourceVolumeMode:
+                description: SourceVolumeMode is the mode of the volume whose snapshot
+                  is taken. Can be either “Filesystem” or “Block”. If left empty,
+                  will be treated as “Unknown”.
```

Comment:
I feel we need to clarify the implications when it is treated as "Unknown". Also gives an impression that "Unknown" is one of the supported values, while it is not.

Reply:
Maybe just rephrase it: "If left empty, sourceVolumeMode is unknown."

Reply:
Done